Makefile

```diff
@@ -103,10 +103,13 @@ SCHED_GETCPU =
 ## much of a performance penalty
 ALLOC_SANITY = -DALLOC_SANITY=0
 
-## Enable hooking of memcpy to detect out of bounds r/w
-## operations on chunks allocated with IsoAlloc. Does
-## not require ALLOC_SANITY is enabled
-MEMCPY_SANITY = -DMEMCPY_SANITY=0
+## Enable hooking of memcpy/memset to detect out of bounds
+## r/w operations on chunks allocated with IsoAlloc. Does
+## not require ALLOC_SANITY is enabled. On MacOS you need
+## to set FORTIFY_SOURCE to 0. Leave these commented if
```

Comment:
`FORTIFY_SOURCE=0` is highly problematic, should a bug be opened to tackle this?

Reply:
Yes. Remove it and compile IsoAlloc for MacOS to see the build error. Ideally we should only do this for MacOS where its actually breaking the build. And then probably another bug to figure out why its breaking in the first place.

Reply:
Fixed here https://github.com/struct/isoalloc/commit/f6b38d28c2e0928f420f1dbca18d9ff6b6894f7a

Reply:
It still sucks: isoalloc shouldn't disable `FORTIFY_SOURCE` :/

Reply:
I agree. Just need time to run down why the build is breaking on MacOS